spec/form_models/jobseekers/job_alert_further_feedback_form_spec.rb

```diff
@@ -3,4 +3,23 @@
 RSpec.describe Jobseekers::JobAlertFurtherFeedbackForm, type: :model do
   it { is_expected.to validate_presence_of(:comment) }
   it { is_expected.to validate_length_of(:comment).is_at_most(1200) }
+  it {
+    is_expected.to validate_inclusion_of(:user_participation_response)
+                  .in_array(Feedback.user_participation_responses.keys)
+  }
+
+  describe "#email" do
+    context "when the jobseeker is interested in participating in research" do
+      before { allow(subject).to receive(:user_participation_response).and_return("interested") }
+
+      it { is_expected.to validate_presence_of(:email) }
+      it { is_expected.to_not allow_value("invalid@email@com").for(:email) }
+    end
+
+    context "when the jobseeker is uninterested in participating in research" do
```

Comment:
not sure how useful is this spec. I would remove it, otherwise to be consistent we should do it everywhere there is conditional validation